lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2043,6 +2043,73 @@ bool LLVMToSPIRVBase::transBuiltinSet() {
   return true;
 }
 
+/// Transforms SPR-IR work-item builtin calls to SPIRV builtin variables.
+/// e.g.
+///  SPR-IR: @_Z33__spirv_BuiltInGlobalInvocationIdi(i)
+///    is transformed as:
+///  x = load GlobalInvocationId; extract x, i
+/// e.g.
+///  SPR-IR: @_Z22__spirv_BuiltInWorkDimi()
+///    is transformed as:
+///  load WorkDim
```

Comment:
We usually call this representation SPV-IR, right?
```suggestion
/// Transforms SPV-IR work-item builtin calls to SPIRV builtin variables.
/// e.g.
///  SPV-IR: @_Z33__spirv_BuiltInGlobalInvocationIdi(i)
///    is transformed as:
///  x = load GlobalInvocationId; extract x, i
/// e.g.
///  SPV-IR: @_Z22__spirv_BuiltInWorkDim()
///    is transformed as:
///  load WorkDim
```
(also corrected mangling).

Reply:
Typo fixed.